modules/end-user-guide/partials/assembly_creating-a-workspace-from-included-options.adoc

```diff
@@ -0,0 +1,20 @@
+:parent-context-of-creating-a-workspace-from-included-options: {context}
+
+[id="creating-a-workspace-from-included-options_{context}"]
+= Creating a workspace from included options
```

Comment:
This sentence used to be *Creating a workspace from code sample*.
There are two sub-sections, one of which I named *Creating a workspace from a code sample*.
The other sub-section I named *Creating a workspace from a template devfile*.
This means we cannot keep the top-level heading *Creating a workspace from code sample* because it does not cover both sub-sections and because in such case the same heading would be duplicated at two levels.
As a solution, I am proposing a new top-level heading here:
*Creating a workspace from a code sample* becomes *Creating a workspace from included options*, followed by the intro sentence: _{prod-short} includes optional code samples and devfile templates to simplify a workspace creation._
This should meet the needs of the first-time user who needs to understand two basic ideas:
1. Che has such *included* features as codes samples and devfile templates to simplify a workspace creation.
2. These *included* features are optional, hence *included options*.

Reply:
Or some other heading as an alternative.

Reply:
Or we elevate the second heading (**Creating a workspace from a template devfile**).

Reply:
Sounds like an idea we could all agree on. Nice. GO fo it :)